### PR TITLE
docs: init book

### PR DESCRIPTION
**Pull request by hi-rustin**

### CODE REVIEW

File: docs/book.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["hi-rustin"]
+language = "en"
+multilingual = false
+src = "src"
+title = "devstats in TiDB"
```
Comment (Mini256) [inline docs/book.toml:6]:
```suggestion
title = "DevStats in TiDB"
```

**Review by Mini256 — COMMENTED** [review]
Why we need the project folders?

**Comment by Mini256** [comment]
I still don’t understand why we need the project folders?

**Comment by hi-rustin** [comment]
> I still don’t understand why we need the project folders?



> I still don’t understand why we need the project folders?

Maybe we'll create special  dashboards for projects?

**Comment by Mini256** [comment]
> Maybe we'll create special dashboards for projects?

Maybe you should put the project directory in the dashboards directory:
https://github.com/cncf/devstats/tree/master/docs/dashboards

**Review by Mini256 — APPROVED** [review]

**Pull request merged**